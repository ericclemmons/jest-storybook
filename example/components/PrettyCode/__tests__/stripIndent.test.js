import stripIndent from "../stripIndent";

describe("stripIndent", () => {
  it("should remove indentation from blocks", () => {
    expect(stripIndent(`
      <body>
        <section>
          <nav />
        </section>
      </body>
    `)).toMatchSnapshot();
  });
});
