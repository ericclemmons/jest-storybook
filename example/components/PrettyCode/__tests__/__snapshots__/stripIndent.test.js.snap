exports[`stripIndent should remove indentation from blocks 1`] = `
"<body>
  <section>
    <nav />
  </section>
</body>"
`;
